gnome-image-installer/pages/install/gis-scribe.c

```diff
@@ -41,6 +41,7 @@
 typedef enum {
   GIS_SCRIBE_TASK_TEE        = 1 << 0,
   GIS_SCRIBE_TASK_VERIFY     = 1 << 1,
+  GIS_SCRIBE_TASK_DECOMPRESS = 1 << 2,
```

Comment:
Ah, this explains the `1 << 3` in a previous commit. Ignore my other comment.

Reply:
:)